src/planner/vectorfield/MoveHandStraightVectorField.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef AIKIDO_PLANNER_VECTORFIELD_MOVEHANDSTRAIGHTVECTORFIELD_H_
+#define AIKIDO_PLANNER_VECTORFIELD_MOVEHANDSTRAIGHTVECTORFIELD_H_
+#include <Eigen/Geometry>
+#include <aikido/planner/vectorfield/VectorFieldPlanner.hpp>
+
+namespace aikido {
+namespace planner {
+namespace vectorfield {
+
+class MoveHandStraightVectorField {
```

Comment:
Nit: This style inconsistent should be fixed once this file is moved to `include` directory and `make format` is executed.